README.md

```diff
@@ -1,26 +1,34 @@
+- Observable runtime. The agent runtime state and knowledge of the world can be monitored at all times, with different levels of detail. Human readable metadata for tasks can be provided via the task `description` property. Plug in a logger to generate human readable logs.
+- Parallel execution of tasks. The planner automatically detects when operations can be performed in parallel and creates branches in the plan to tell the agent to run branches concurrently.
+- Easy to debug. Agent observable state and known goals allow easy replicability when issues occur. The planning decision tree and resulting plans can be diagrammed to visually inspect where planning is failing.
 
-## Concepts
+## Requirements
+
+- Node.js 18+. Other runtimes are unsupported at this moment.
+
+## Installation
+
+```
+npm install --save mahler
+```
+
+## Core Concepts
 
 - **Autonomous system agent** a process on a system that needs to operate with little or not feedback from the outside world. An autonomous agent needs to be able to recover from failures and adapt if conditions on the system change while performing its duties. In our definition, such an agent operates based on a given goal and will keep trying to achieve the goal until the goal changes or some other exit conditions are met.
 - **Hierarchical Task Network (HTN)** is a type of automated planning system that allows to define actions in the planning domain in a hierarchical manner, allowing actions to be re-used as part of compound tasks. This reduces the search domain and provides developers more control on what plans are preferable (over something like [STRIPS](https://es.wikipedia.org/wiki/STRIPS)). This has made this type of system popular in [game design](https://www.youtube.com/watch?v=kXm467TFTcY).
-- **Task** a task is any operation defined by a domain expert to provide to the planner. A task can be an _action_, e.g. "download a file", "write X to the database", or a _method_, i.e. a compound task, that provides a sequence of steps to follow.
-- **Action Task** an action task is defined by a _condition_, i.e. some predicate on the system state that needs to be true before the task can be chosen, an _effect_, i.e. a transformation on the state that will result from running the action, and an action function, which is the actual operation that will be executed by the plan runner when the task is chosen.
-- **Plan** a plan is a sequence of actions to execute in order to achieve a certain goal.
-- **Goal** a goal is a desired state of the system. e.g, "temperature of the room == 25 degrees".
+- **Task** a task is any operation defined by a domain expert to provide to the planner. A task can be a primitive task, called an _action_ on this framework, e.g. "download a file", "write X to the database", or a _method_, i.e. a compound task, that provides a sequence of steps to follow.
```

Comment:
`, called an` -> `called an`

Reply:
This one I'm not so sure about, it seems like the pause of the comma is needed here